src/components/picker/index.js

```diff
@@ -336,5 +340,11 @@ class Picker extends BaseComponent {
   }
 }
 
+const styles = StyleSheet.create({
```

Comment:
this is a general comment on the PR. 
not sure fixing the Picker is the right solution in this case. 
Picker is based on TextField, therefore the color, and the UI in general, should be derived from the TextField. 
Now, I've checked if the TextField has the same issue and it does. so if anything we should fix the source of the problem. 

Having said that, it's possible the fix should part of our the private uilib since maybe our color configuration is there. 
There's currently an open PR by Inbal in the private that is related to the TextField. 
so let's put this one on hold till we migrate her PR and then I would recommend investigate the issue in the public code.

Reply:
We can put this on hold, but I think the solution should be for the public lib as well since I think it's a bad UX in general, not just for us.